tests/data/profile_stability/rhel8/stig.profile

```diff
@@ -363,6 +363,7 @@ selections:
 - set_password_hashing_algorithm_passwordauth
 - set_password_hashing_algorithm_systemauth
 - set_password_hashing_min_rounds_logindefs
+- ssh_keys_passphrase_protected
```

Comment:
It was probably a change in some script used in the build process, outside the scope of this PR. I noticed that this line was simply repositioned alphabetically.

Reply:
Good catch Vojta.
I see the new rule was added by this PR https://github.com/ComplianceAsCode/content/pull/10017, but it did not add the rule to stabilization.

@marcusburghardt Could you make another PR for that?

Reply:
I can, but we would need to review and merge it before this one. Because it will need to be rebased.
@yuumasato , this line is not new in the files. It is only in a different position.

Reply:
I mean, if you just remove those changes related to ssh_rule from this PR (so that changes are related only to the PAM flag), nothing should break. Or am I missing something?

Reply:
But the baseline for the stability test will still be broken. It needs to be updated anyways. Because that I said it should be fixed then this PR should be rebased if we want to do it in a separate PR. But reinforcing, the change is simply about the line position, nothing added or removed around the ssh line. What is the issue of keeping it? I think the effort to separate this in a different PR and rebasing this one is not worth unless we have a real issue with that.

Reply:
I don't understand why a rebase would be required.

Reply:
I agreed to manually update the baseline for the STIG stability checks in the last commit in order to help @yuumasato to accelerate some offline tests based on a offline conversation we had. However, the #10182 must me merged as soon as possible to keep the STIG stability checks consistent in Upstream.